libgearman-server/server.cc

```diff
@@ -807,15 +807,16 @@ gearmand_error_t gearman_server_run_command(gearman_server_con_st *server_con,
       gearman_server_job_st *server_job= gearman_server_job_get(Server,
                                                                 (char *)(packet->arg[0]), (size_t)strlen(packet->arg[0]),
                                                                 server_con);
-      gearmand_log_debug(GEARMAN_DEFAULT_LOG_PARAM,
-                         "Exception being sent from: %.*s(%lu)",
-                         server_job->function->function_name_size, server_job->function->function_name, server_job->function->function_name_size);
       if (server_job == NULL)
       {
         return _server_error_packet(GEARMAN_DEFAULT_LOG_PARAM, server_con, GEARMAN_JOB_NOT_FOUND, 
-                                    gearman_literal_param("An exception was received for a job that does not exist"));
+                                  gearman_literal_param("An exception was received for a job that does not exist"));
```

Comment:
Is this whitespace change intentional?

Reply:
No. The change is revoked https://github.com/gearman/gearmand/pull/183/commits/991db14972411b2288e90b80c725a74b24c47624